README.md

```diff
@@ -140,6 +140,7 @@ You can build and/or use the maven coordinates to work with `azure-cosmosdb-spar
 
 | Spark | Scala | Latest version |
 |---|---|---|
+| 2.4.0 | 2.11 | [azure-cosmosdb-spark_2.4.0_2.11_3.0.3](https://search.maven.org/remotecontent?filepath=com/microsoft/azure/azure-cosmosdb-spark_2.4.0_2.11/3.0.3/azure-cosmosdb-spark_2.4.0_2.11-3.0.3-uber.jar)
 | 2.4.0 | 2.11 | [azure-cosmosdb-spark_2.4.0_2.11_2.1.2](https://repo1.maven.org/maven2/com/microsoft/azure/azure-cosmosdb-spark_2.4.0_2.11/2.1.2/azure-cosmosdb-spark_2.4.0_2.11-2.1.2-uber.jar)
```

Comment:
Please remove this link (only the latest for each Spark version is supposed to be listed here)

Reply:
Ok, i will update it.